src/System/Gr/GrScreen.ZC

```diff
@@ -34,7 +34,7 @@ U0 GrUpdateTaskWin(CTask *task)
 			if (gr.fp_wall_paper)
 				(*gr.fp_wall_paper)(task);
 		}
-		else if (!(task->win_inhibit & WIF_SELF_DOC))
+		/* else */ if (!(task->win_inhibit & WIF_SELF_DOC))
```

Comment:
I haven't noticed any bug from doing this, however the wallpaper won't display without it.
I've decided to modify the GrScreen instead of asking the user to do it.